### PR TITLE
Add details how to enable Actions in new appliance

### PR DESCRIPTION
**Pull request by kgrzebien**
When we restore in new appliance from backup that includes Actions we must configure it beforehand. We must configure external storage and enable Actions. If we don't do it `ghe-restore` process will fail. 

This is to add link to documentation with necessary setup details.

Related to https://github.com/github/c2c-actions-platform/issues/4293

### CODE REVIEW

File: bin/ghe-restore [hunk 1]
```diff
@@ -258,6 +258,7 @@ fi
 if [ -d "$GHE_RESTORE_SNAPSHOT_PATH/mssql" ] || [ -d "$GHE_RESTORE_SNAPSHOT_PATH/actions" ]; then
   if ! ghe-ssh "$GHE_HOSTNAME" -- 'ghe-config --true app.actions.enabled'; then
     echo "Error: $GHE_HOSTNAME must have GitHub Actions enabled before restoring since the snapshot contains Actions data. Aborting." 1>&2
+    echo "Setup details for enabling Actions can be found here: https://docs.github.com/en/enterprise-server@3.3/admin/github-actions/advanced-configuration-and-troubleshooting/backing-up-and-restoring-github-enterprise-server-with-github-actions-enabled" 1>&2
```
Comment (zarenner) [inline bin/ghe-restore:261]: Leave out specific version (3.3)

Comment (kgrzebien) [reply bin/ghe-restore]: Thanks, I paste wrong URL.

File: bin/ghe-restore [hunk 2]
```diff
@@ -258,6 +258,7 @@ fi
 if [ -d "$GHE_RESTORE_SNAPSHOT_PATH/mssql" ] || [ -d "$GHE_RESTORE_SNAPSHOT_PATH/actions" ]; then
   if ! ghe-ssh "$GHE_HOSTNAME" -- 'ghe-config --true app.actions.enabled'; then
     echo "Error: $GHE_HOSTNAME must have GitHub Actions enabled before restoring since the snapshot contains Actions data. Aborting." 1>&2
+    echo "Setup details for enabling Actions can be found here: https://docs.github.com/en/enterprise-server/admin/github-actions/advanced-configuration-and-troubleshooting/backing-up-and-restoring-github-enterprise-server-with-github-actions-enabled" 1>&2
```
Comment (zarenner) [inline bin/ghe-restore:261]: Can we provide a version-specific link, since we have access to the appliance to get the version?

**Review by zarenner — APPROVED** [review]

File: bin/ghe-restore (hunk 2 above)
Comment (kgrzebien) [reply bin/ghe-restore]: Version specific has been added to documentation URL. 👍

**Pull request merged**